compiler/rustc_session/src/config.rs

```diff
@@ -2093,6 +2057,34 @@ fn collect_print_requests(
     prints
 }
 
+fn check_print_request_stability(
+    early_dcx: &EarlyDiagCtxt,
+    unstable_opts: &UnstableOptions,
+    (print_name, print_kind): (&str, PrintKind),
+) {
+    match print_kind {
+        PrintKind::AllTargetSpecs | PrintKind::CheckCfg | PrintKind::TargetSpec
+            if !unstable_opts.unstable_options =>
+        {
+            early_dcx.early_fatal(format!(
+                "the `-Z unstable-options` flag must also be passed to enable the `{print_name}` \
```

Comment:
I added backticks to the `print_name` to distinguish from the surrounding text, but just a style thing